sceptre/template.py

```diff
@@ -110,11 +110,8 @@ def _call_sceptre_handler(self):
 
         try:
             body = module.sceptre_handler(self.sceptre_user_data)
-        except AttributeError:
-            raise TemplateSceptreHandlerError(
-                "The template does not have the required "
-                "'sceptre_handler(sceptre_user_data)' function."
-            )
+        except AttributeError as e:
+            raise e
```

Comment:
I think @jamesroutley meant to keep the if/else statement but instead of raising:
```
raise TemplateSceptreHandlerError("Error generating the template: {}".format(e.message))
````
just raise the existing Exception 
```
raise e
```